app/views/challenges/48in24.html.haml

```diff
@@ -1,11 +1,12 @@
 - week = ((Time.zone.today - Date.new(2024, 1, 15)) / 7).ceil
-- featured_exercise = @exercises.find { |e| e.week == week }
+- exercises = @exercises.select { |e| e.week <= week }
+- tracks = Track.where(slug: exercises.flat_map(&:featured_tracks)).pluck(:slug, :title).to_h
+
+- featured_exercise = exercises.last
 - if featured_exercise
   - generic_exercise = GenericExercise.for(featured_exercise.slug)
-- live_exercises = @exercises.select { |e| e.week <= week }
-- flat_live_exercises = live_exercises.flat_map(&:featured_tracks).uniq
-- featured_tracks = Track.where(slug: flat_live_exercises).order(:title).pluck(:slug, :title).to_h
-- num_placeholders = 48 - live_exercises.size
+  - featured_tracks = Track.where(slug: featured_exercise.featured_tracks).pluck(:title).sort
```

Comment:
Could you check to see if this works? It saves a query
```suggestion
  - featured_tracks = featured_exercise.featured_tracks.select{|slug| tracks[slug]}.sort
```

Reply:
Yes, this works. :)